lib/otel/transformation-rules.json

```diff
@@ -968,6 +972,39 @@
       }
     }
   },
+  {
```

Comment:
need this rule atm because right now the gcp producer spans hit the fallback producer because it lacks `message.operation`. [5.1.0](https://github.com/googleapis/nodejs-pubsub/pull/2039) will address this and we can remove this rule